ServerCore/Areas/Identity/Pages/Account/ConfirmEmail.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@page
```

Comment:
Should this page be hidden/removed?

Reply:
We're going to need this at some point, I don't think there are any links to it at the moment.